app/models/vita_partner.rb

```diff
@@ -50,7 +50,6 @@ class VitaPartner < ApplicationRecord
   accepts_nested_attributes_for :source_parameters, allow_destroy: true, reject_if: ->(attributes) { attributes['code'].blank? }
 
   def self.client_support_org
-    # When a person messages us, but their contact info does not match any Client, link them to this org.
```

Comment:
I think we use this organization for other things, so I'm not going to remove it. Does that sound right @alaroia-cfa

Reply:
@bytheway875 I think that this org in the codebase only exists for the purpose of creating unlinked clients. I did a search for the method and that seems to agree. I think it's peaceful to delete `VitaPartner.client_support_org` from the codebase.

50/50 chance it's peaceful to delete the actual database record in prod. Check with Yvonne or client success and/or query this org to see if it's being used (look for login-able clients assigned to this org maybe?). There's a chance that the record is used for other things by Client Success but also a decent chance it's not.

Reply:
ummm... interesting enough, this record does not exist in production. so i dont know hwo the unlinked clients thing was working.